installer/PowerToysSetupCustomActions/CustomAction.cpp

```diff
@@ -76,7 +75,7 @@ UINT __stdcall ApplyModulesRegistryChangeSetsCA(MSIHANDLE hInstall)
     hr = getInstallFolder(hInstall, installationFolder);
     ExitOnFailure(hr, "Failed to get installFolder.");
 
-    for (const auto& changeSet : getAllModulesChangeSets(installationFolder, false))
+    for (const auto& changeSet : getAllModulesChangeSets(installationFolder, true))
```

Comment:
if nobody ever calls `getAllModulesChangeSets` with differnet values for this bool, should we just remove it?